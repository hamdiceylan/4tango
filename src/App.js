import React from 'react';
import { Route, Switch } from 'react-router-dom';
import PropTypes from 'prop-types';
import CssBaseline from '@material-ui/core/CssBaseline';
import { withStyles } from '@material-ui/core/styles';
import Footer from './app/layout/Footer/Footer';
import Header from './app/layout/Header/Header';
import HomePage from './features/HomePage/HomePage'

const styles = theme => ({
  appBar: {
    position: 'relative',
  },
  icon: {
    marginRight: theme.spacing.unit * 2,
  },
  layout: {
    width: 'auto',
    marginLeft: theme.spacing.unit * 3,
    marginRight: theme.spacing.unit * 3,
    [theme.breakpoints.up(1100 + theme.spacing.unit * 3 * 2)]: {
      width: 1100,
      marginLeft: 'auto',
      marginRight: 'auto',
    },
  },
  cardGrid: {
    padding: `${theme.spacing.unit * 8}px 0`,
  },
  card: {
    height: '100%',
    display: 'flex',
    flexDirection: 'column',
  },
  cardMedia: {
    paddingTop: '56.25%', // 16:9
  },
  cardContent: {
    flexGrow: 1,
  },
  footer: {
    backgroundColor: theme.palette.background.paper,
    padding: theme.spacing.unit * 6,
  },
});


function App(props) {
  const { classes } = props;

  return (
    <React.Fragment>
      <CssBaseline />
      <Header classes={classes}/>
      <div>
        <Switch>
          <Route exact path='/' component={() => <HomePage styles={styles} classes={classes} />}/>
        </Switch>
        <Route 
          path="/(.+)"
          render ={() => (
            <div>
                <Switch>
                  {/* <Route path='/login' component={EventDetail}/> */}
                </Switch>
            </div>
          )}
          />
      </div>
      <Footer styles={styles.footer} />
    </React.Fragment>
  );
}

App.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(App);




