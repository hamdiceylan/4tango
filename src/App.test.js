import React from 'react';
import App from './App';
import { shallow } from 'enzyme'
import enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
enzyme.configure({ adapter: new Adapter() });


describe('renders without crashing', () => {
  const app = shallow(<App />);

  it('renders properly', () =>{
    expect(app).toMatchSnapshot();
  })
  
});
