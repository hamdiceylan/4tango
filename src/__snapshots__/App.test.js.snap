// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing renders properly 1`] = `ShallowWrapper {}`;
